compiler-rt/lib/sanitizer_common/sanitizer_common_interceptors.inc

```diff
@@ -6304,10 +6304,38 @@ INTERCEPTOR(int, fclose, __sanitizer_FILE *fp) {
 #endif
 
 #if SANITIZER_INTERCEPT_DLOPEN_DLCLOSE
+// Returns 1 if 'suffix' matches the end of 'str', 0 otherwise
+static int internal_is_suffix(const char *suffix, const char *str) {
```

Comment:
I guess we usually use `internal_<some libc function>` 

This is a new function, so we can use Camel style with bool as return.
And usually we search the second argument in the first.

Reply:
Function was removed since the new approach uses exact string matching.